client/me/purchases/cancel-privacy-protection/index.jsx

```diff
@@ -70,8 +71,7 @@ const CancelPrivacyProtection = React.createClass( {
 			return true;
 		}
 
-		const { selectedSite } = props,
-			purchase = getPurchase( props );
+		const { selectedSite } = props, purchase = getPurchase( props );
```

Comment:
Would it be possible to keep every variable in its own line as we usually do it?